src/lib/jsonParse/jsonRequest.cpp

```diff
@@ -198,14 +198,13 @@ std::string jsonTreat
 
   if (reqP == NULL)
   {
-    std::string  details = std::string("Sorry, no request treating object found for RequestType /") + requestType(request) + "/";
     std::string  errorReply;
     char         reqTypeV[STRING_SIZE_FOR_INT];
 
-    restErrorReplyGet(ciP, SccBadRequest, details, &errorReply);
+    restErrorReplyGet(ciP, SccBadRequest, "service not found", &errorReply);
```

Comment:
Maybe "service not found" should be a #define? I remember we have a .h with the "magic strings" used in error messages.

Reply:
Fixed in 4dfd0c0